### ISSUE TITLE
cgroup2 support

### ISSUE THREAD

**Issue by hakavlad** [poelzi/ulatencyd#67]
cgroups with systemd 243 by default mounted only as unified hierarchy.

I would like the daemon to start supporting the cgroup2.

See also https://systemd.io/CGROUP_DELEGATION.html

**Comment by hakavlad** [comment]
This excellent demon can live for a long time if it learns to be friends with systemd and cgroup2.

**Comment by poelzi** [comment]
yes. please become the new maintainer, I would love to have it running again but seriously don't find the time required to derot the code.
